src/flows/braze/delete_users_with_external_id_without_email_or_aliases_flow.py

```diff
@@ -0,0 +1,127 @@
+from prefect import Flow, context, task, Parameter
+import datetime
+import re
+from typing import Any, Dict, List, Tuple
+from dataclasses import dataclass
+
+from api_clients.pocket_snowflake_query import OutputType, PocketSnowflakeQuery
+from common_tasks.mapping import split_in_chunks
+from utils import config
+from utils.flow import get_s3_result, get_flow_name, get_cron_schedule
+from api_clients.braze import models
+from api_clients.braze.client import (
+    BrazeClient,
+    IDENTIFY_USER_ALIAS_LIMIT
+)
+
+from utils.iteration import chunks
+
+FLOW_NAME = get_flow_name(__file__)
+
+EXTRACT_QUERY = """
+SELECT
+    BRAZE_ID
+FROM "{table_name}"
+WHERE email is null 
```

Comment:
Are we going to write a different flow to delete ghost accounts?

Reply:
@kirill-demtchouk I think this flow should handle that case?